src/components/Icon/Icon.types.ts

```diff
@@ -9,11 +9,11 @@ export enum IconSize {
     XSmall = '14px',
 }
 
-export interface IconProps extends OcBaseProps<HTMLSpanElement>, MdiIconProps {
+export interface IconProps extends OcBaseProps<HTMLSpanElement> {
```

Comment:
Can we take two different props approach here.
https://stackoverflow.com/questions/58630750/react-typescript-component-with-two-different-prop-interfaces